java/arcs/core/analysis/PolicyVerifier.kt

```diff
@@ -89,52 +89,29 @@ class PolicyVerifier(val options: PolicyOptions) {
             ?.let { claims -> InformationFlow.IngressInfo(handleNode, claims) }
     }
 
-    /**
-     * Returns the egress checks.
-     *
-     * @throws [PolicyViolation] if recipe violates egress requirements.
-     */
-    private fun getEgressChecks(
-        policy: Policy,
-        recipe: Recipe,
-        egressCheck: Predicate
-    ): Map<ParticleSpec, List<Check>> {
-        // Check that egress particles are compliant with policy.
-        val egressParticles = recipe.particles.filterNot { it.spec.isolated }
-        checkEgressParticles(policy, egressParticles)
-
-        // Create a check for each handle connection needs in the egress particles.
-        return egressParticles.associate { particle ->
-            val checks = particle.spec.connections.values
-                .filter {
-                    // TODO(b/157605232): Also check canQuery -- but first, need to add QUERY to
-                    // the Direction enum in the manifest proto.
-                    it.direction.canRead
-                }
-                .map { connectionSpec ->
-                    Check.Assert(AccessPath(particle, connectionSpec), egressCheck)
-                }
-            particle.spec to checks
-        }
-    }
-
     /**
      * Verifies that the given egress particle nodes match the policy. The only egress particles
      * allowed to be used with a policy are in [options.policyEgresses] map.
```

Comment:
Comment is stale as Silvano pointed out elsewhere.

Reply:
Done